packages/mobile/src/send/saga.ts

```diff
@@ -77,7 +79,7 @@ export function* watchQrCodeDetections() {
 
     if (action.scanIsForSecureSend) {
       secureSendTxData = action.transactionData
-      requesterAddress = action.requesterAddrress
+      requesterAddress = action.requesterAddress
```

Comment:
This is the bug I initially spotted while checking our QR code handling.

Reply:
Good catch. This bug would result in secure send failing for a payment request from an unverified account that was being validated via QR code.